public/locales/en/translations.json

```diff
@@ -402,6 +403,25 @@
           "value": "Value",
           "valuePlaceholder": "Enter value"
         },
+        "actions":{
+          "actions": "Actions",
+          "action": "Action",
+          "addAction": "Add action",
+          "select": "Choose an action",
+          "selectionPlaceholder": "Select action",
+          "emailAddresses": "Email addresses",
+          "phoneNumbers": "Phone numbers",
+          "enterEmail": "Enter email of notification recipients",
+          "enterPhoneNumber": "Enter phone number of notification recipients",
+          "emailComments": "Email comments",
+          "smsComments": "Message comments",
+          "enterEmailComm": "Enter comments for the email recipient",
+          "enterSMSComm": "Enter comments for the text recipient",
+          "syntaxError": "Invalid syntax",
```

Comment:
Is "Invalid syntax" being shown to the user? is there friendlier verbiage we could use? like "Not a valid email address"?